sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/parallel_for.py

```diff
@@ -82,7 +82,7 @@ def __init__(
             self._outputs = self._build_outputs_dict_without_meta(outputs=actual_outputs)
 
         self._items = items
-        self._validate_items(raise_error=True)
+        # self._validate_items(raise_error=True)
```

Comment:
so we'll put off items validation? then why keep this line?

Reply:
Good catch, forgot to remove this..